packages/fxa-auth-server/lib/senders/templates/downloadSubscription.html

```diff
@@ -1,27 +1,114 @@
-{{> icon}}
+<table
```

Comment:
HTML email template hurt my brain. But, this markup is taken from the `subplatEmails.zip` exported from Marketing Cloud attached to the email epic https://jira.mozilla.com/browse/FXA-1147